src/ILVerify/src/ILImporter.StackValue.cs

```diff
@@ -55,14 +58,24 @@ static public StackValue CreateUnknown()
             return new StackValue(StackValueKind.Unknown);
         }
 
-        static public StackValue CreatePrimitive(StackValueKind kind)
+        static public StackValue CreatePrimitive(TypeDesc type)
```

Comment:
(It should also keep the ILVerify code more similar to the other importers that do not verify.)